src/lib/auth.ts

```diff
@@ -79,7 +79,11 @@ export const ROLES: {
 		'resin.application_environment_variable.all',
 		'resin.application_tag.all',
 		'resin.application_type.all',
-		'resin.device.all',
+		'resin.device.create',
+		'resin.device.read',
+		'resin.device.update',
+		'resin.device.delete',
+		`resin.device.tunnel-any`,
```

Comment:
Why not keep it as `resin.device.all` rather than manually list all current possible permissions?

Reply:
@Page- I was, as you, under the impression that `all` only granted CRUD actions, I thought it was literally automatically "expanded" by Pine to permissions for this set of 4 actions. But @richbayliss found this is not the case and `all` actually grants all permissions, hence this PR. IMO the first behaviour is more intuitive and I wonder if this is a bug with Pine or just wrong expectations (that we all 3 had :).

Reply:
No, I was always fully aware that all means all, otherwise it'd be something like `crud`/`all-http`/etc, otherwise how would you name one that is actually all permissions? "no-really-I-actually-mean-all"? :P

Reply:
@Page- @dfunckt in this instance, `resin.device.all` would be accurate, but we should therfore drop the `resin.device.tunnel-22222` as this makes it look like only 22222 would be allowed. Agreed?

Reply:
Done